pages/getting-started/basic-tutorial.html

```diff
@@ -983,8 +983,8 @@ <h4>Delete Entity with ETag</h4>
 <div class="col-1-12"></div>
 <div class="col-1-4" id="tutorial-contents">
 
-<nav class="tutorial-sidebar affix" aria-label="Table of contents" >
-<ul class="nav tutorial-sidenav">
+<nav class="tutorial-sidebar affix" role="complementary" aria-label="Page Contents" >
+<ul class="nav tutorial-sidenav" role="navigation">
```

Comment:
```suggestion
<ul class="nav tutorial-sidenav">
```
Nav already communicates the role is navigation.

Reply:
Removed extra attributes.